app/services/cluster_stats/update_epoch_duration_service.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module ClusterStats
+  class UpdateEpochDurationService
+
+    AVG_EPOCH_SIZE = 5
```

Comment:
What does this constant represent? It's not epoch size, all the more average epoch size. It's a number of epochs that we pull for calculations, so should be called appropriately, NUMBER_OF_EPOCHS.